src/Console/Application.php

```diff
@@ -71,6 +78,18 @@ public function doRun(InputInterface $input, OutputInterface $output)
             : ($input->hasParameterOption('--format', true) && 'txt' !== $input->getParameterOption('--format', null, true) ? null : $output)
         ;
         if (null !== $stdErr) {
+            if ($this->deprecationWarningsEnabled) {
+                $previousErrorHandler = set_error_handler(function ($severity, $message, $file, $line) use (&$previousErrorHandler, $stdErr) {
```

Comment:
one can programatically call `doRun` multiple times, each time he would override error handler with next and next lambda we provides here.
we are also never resetting the error handler to original one if it was there.

Reply:
Should be fixed.

Reply:
ack, will look on that